lib/floe/workflow/intrinsic_function/transformer.rb

```diff
@@ -151,6 +151,11 @@ def check_types(args, function, signature)
           OpenSSL::Digest.hexdigest(algorithm, data)
         end
 
+        rule(:states_json_to_string => {:args => subtree(:args)}) do
+          args = Transformer.process_args(args(), "States.JsonToString", [Object])
```

Comment:
If it's Object then we should have a separate test for each type of object possible that can be converted to JSON (might be useful to run this through the stepfunctions simulartor to see what it does - my script in #244 is helpful for this purpose)